### PR TITLE
Master -> main branch name change in vets-website (#40469)

### PR DESCRIPTION
**Pull request by alexandec**
## Description of change
We will be renaming the vets-website `master` branch `main`. This PR updates vets-website references in this repo accordingly.

**DO NOT MERGE UNTIL DEFAULT BRANCH NAMES OF VETS-WEBSITE AND CONTENT-BUILD HAVE BEEN UPDATED**.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40469

### CODE REVIEW

**Review by acrollet — DISMISSED** [review]
Approved for check-in changes.

**Review by jhwuz — CHANGES_REQUESTED** [review]
Can you merge in master or rebase? The specs seem to be failing due to a poppler error that has been fixed in the current version of master.

**Review by ericboehs — CHANGES_REQUESTED** [review]
Please move all DS_Store files.

File: .gitignore [hunk 1]
```diff
@@ -84,3 +84,6 @@ Brewfile.lock.json
 
 # Ignore Test User Dashboard csv
 test_users.csv
+
+# Ignore Mac OS hidden file
+.DS_Store
```
Comment (ericboehs) [inline .gitignore:89]: OS level files should be ignored in your global gitignore, not per repository. See the "Felt tip" here: https://stackoverflow.com/a/19299889/282720.

**Review by ericboehs — CHANGES_REQUESTED** [review]
See comment on .gitignore.

File: .gitignore (hunk 1 above)
Comment (it-harrison) [reply .gitignore:89]: thanks for the tip!

**Review by jhwuz — APPROVED** [review]

**Review by ericboehs — APPROVED** [review]

**Pull request merged**